src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawDriverAdapter.java

```diff
@@ -26,12 +28,16 @@ public void operateTo(int x, int y) {
 		ILine line = LineFactory.getBasicLine();
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
-
-		drawLine(line);
+		
+		this.startX = line.getEndCoordinateX();
+		this.startY = line.getEndCoordinateY();
+		
+		DrawerFeature.getDrawerController().drawLine(line);
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.